triggers/service/auth/src/main/scala/com/daml/auth/middleware/api/Client.scala

```diff
@@ -24,15 +24,18 @@ import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.unmarshalling.Unmarshal
 import com.daml.auth.middleware.api.Client.{AuthException, RefreshException}
 import com.daml.auth.middleware.api.Tagged.RefreshToken
+import com.github.benmanes.caffeine.cache.{Cache, Caffeine}
```

Comment:
Can we use `com.daml.caching` instead which wraps the caffeine cache?

Reply:
`com.daml.caching.Cache` does not expose `invalidate`, which we need here.

Reply:
Makes sense, maybe worth considering adding it to `com.daml.caching` at some point but not in this PR.